lib/fabrication/schematic/class_resolver.rb

```diff
@@ -0,0 +1,13 @@
+module Fabrication
+  module Schematic
+    class ClassResolver
+      def self.create(name, parent, options)
+        if options[:class_name]
```

Comment:
It is more complicated than this because we don't necessarily have AS available. We also need to check and see if constantize is even a thing.